redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/handler/CertReplaceInRedBeamsHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.sequenceiq.redbeams.flow.redbeams.start.handler;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import org.slf4j.Logger;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.eventbus.Event;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertReplaceInRedBeamsRequest;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertReplaceInRedBeamsSuccess;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.StartDatabaseServerFailed;
+
+@Component
+public class CertReplaceInRedBeamsHandler extends ExceptionCatcherEventHandler<CertReplaceInRedBeamsRequest> {
+
+    private static final Logger LOGGER = getLogger(CertReplaceInRedBeamsHandler.class);
+
+    @Override
+    public String selector() {
+        return EventSelectorUtil.selector(CertReplaceInRedBeamsRequest.class);
+    }
+
+    @Override
+    protected Selectable defaultFailureEvent(Long resourceId, Exception e, Event<CertReplaceInRedBeamsRequest> event) {
+        return new StartDatabaseServerFailed(resourceId, e);
+    }
+
+    @Override
+    protected Selectable doAccept(HandlerEvent<CertReplaceInRedBeamsRequest> event) {
+        LOGGER.debug("CERT replacement has been started");
+        CertReplaceInRedBeamsRequest data = event.getData();
+        try {
+            LOGGER.debug("Skeleton of cert replacement in RedBeams");
+            return new CertReplaceInRedBeamsSuccess(data.getResourceId());
+        } catch (Exception e) {
+            LOGGER.error("Cannot replace the root CERT, error: ", e);
```

Comment:
Logging can be moved to the `defaultFailureEvent` and the try-catch is not required.